engine/class_modules/sc_death_knight.cpp

```diff
@@ -7535,7 +7535,7 @@ std::string death_knight_t::default_food() const
                             "disabled";
 
   std::string blood_food =  ( true_level > 110 ) ? "mechdowels_big_mech" :
-	                          ( true_level > 100 ) ? "lavish_suramar_feast" :
+	                    ( true_level > 100 ) ? "lavish_suramar_feast" :
```

Comment:
Please ignore this trivial change to formatting, this has no impact on function.